why_postgres.mdx

```diff
@@ -0,0 +1,92 @@
+
+- separation of compute and storage, and bottomless storage.
+- optional automatic scaling up and down depending on the load.
+- high availability via read replicas in multiple availability zones and automatic failover.
+- support for both Postgres wire protocol and SQL over HTTP (for easy access from serverless functions).
+- support for most common Postgres extensions.
+- scale up to 1TB of RAM and 128 vCPUs.
+
+We have designed it to optimize for **reliability**, **scalability**, and **cost efficiency**. A few words about each:
+
+#### Reliability
+
+The Xata Postgres service is using AWS Aurora under the hood, a battle-tested service that has been around for a few years and is used by many very large installations. We have been using AWS Aurora since the initial Xata launch and we have a very [good track record]() in terms of reliability. We are now also have an official [SLA]().
+
+Even on the free tier we offer high-availability and redundancy, with replicas running in multiple availability zones. The failover from the primary instance to a replica is automatic and it's guaranteed to have no data loss. At the storage layer, the data is synchronously replicated to six storage nodes across three availability zones.
+
+#### Scalability
+
+Available instance types in Xata go up to 128 vCPUs and 1TB of RAM. Storage can be scaled up to 128TiB per cluster. That is quite a lot of headroom for vertical scaling.
+
+When it comes to horizontal scaling, the fact that you can create databases and branches instantly and that you can move them between clusters without downtime, opens up several architectural possibilities. For example, you can create a logical database for each of your customers, and place them in different regions. This is possible thanks to our [global control plane and logical databases]().
+
+#### Cost efficiency
+
+Xata achieves cost efficiency for as many use cases as possible by offering a high degree of **optionality**. The innovation of the Xata service is that we make it really easy to move from one type of cluster to another, without downtime. This means that you can easily try your workload on different types of clusters and see which one is the most cost-efficient for you.
+
+At small scale, we offer a free tier that includes high-availability and read replicas, no cold-starts, and no instance-pausing-after-a-few-days. We can offer this sustainably because the free tier makes use of [shared clusters]() and auto-scaling.
+
+At large scale, when using [dedicated clusters]() you pay for the resources you use and you have a high degree of control. For example, for predictable load, you can configure a particular instance size, while for unpredictable load, auto-scaling will likely be more cost-efficient. Similarly, you can choose between standard storage and I/O optimized storage, which is more cost-efficient for I/O-heavy workloads.
+
+## The free tier
```

Comment:
I go back and forth on explicitly calling out the free tier. It feels a little like a jab, which is maybe fine. I have some other ideas for the title.

Floating an idea. But what if we closed with this instead and turn it into a call to action. The title would be "Get started on our production ready free tier today" and the content itself would more or less stay the same.

Reply:
Not a bad idea, and I don't see it as a jab. It's an opportunity 😄